akka-actor-typed/src/main/scala/akka/actor/typed/internal/routing/PoolRouterImpl.scala

```diff
@@ -19,14 +19,21 @@ private[akka] final case class PoolRouterBuilder[T](
     poolSize: Int,
     behavior: Behavior[T],
     logicFactory: ActorSystem[_] => RoutingLogic[T] = (_: ActorSystem[_]) => new RoutingLogics.RoundRobinLogic[T],
+    broadcastPredicate: T => Boolean = (_: T) => false,
```

Comment:
A tiny optimization oportunity here is to define a new `ConstantFun.anyToFalse` and then do an instance equality check in `onMessage` so that the broadcast case can be jitted away easier when not used (default).